src/App.css

```diff
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: Montserrat, sans-serif;
+}
+
+button {
+  padding: 1rem;
+  background-color: #0290ff;
+  border-radius: 4px;
+  width: 10rem;
+  height: 3rem;
+  color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: larger;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] The borders on your buttons are still not correct. you should remove them entirely to match the design as the previous reviewer has suggested.